templates/changeDialogLineSpeaker/session.d

```diff
@@ -6,5 +6,6 @@ func int G1CP_{ISSUE_NUM_PAD}_{SHORTNAME}() {
     // ### TODO: Replace OU name of "Xxx_Ou" ###
     // ### TODO: Optional: Switch `other` and `self` ###
```

Comment:
As proposed #370, I would like to pull these grouped comments inline, see other review comment on demo.